src/Aspire.Hosting/Utils/PasswordGenerator.cs

```diff
@@ -0,0 +1,52 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Security.Cryptography;
+
+namespace Aspire.Hosting.Utils;
+
+internal static class PasswordGenerator
+{
+    // Some chars are excluded:
+    // - prevent potential confusions, e.g., 0,o,O and i,I,l
+    // - exclude special chars which could interfere with command line arguments or connection strings, e.g. =,$,...
+
+    internal const string LowerCaseChars = "abcdefghjkmnpqrstuvwxyz"; // exclude i,l,o
+    internal const string UpperCaseChars = "ABCDEFGHJKMNPQRSTUVWXYZ"; // exclude I,L,O
+    internal const string DigitChars = "0123456789";
+    internal const string SpecialChars = "-_#@!./:[]{}+,*()~"; // exclude &<>=;,`'^%$
```

Comment:
The comments say to exclude a comma `,` but it is included in the string.

Reply:
```suggestion
    internal const string SpecialChars = "-_#@!./:[]{}+*()~"; // exclude &<>=;,`'^%$
```